<html>
<head>
<title>PyException_SetCause</title>
<meta name='robots' content='noindex,nofollow'>
<meta name='generator' content='GLOBAL-5.8.1'>
</head>
<body text='#191970' bgcolor='#f5f5dc' vlink='gray'>
<pre>
<a href='../S/2789.html#L300'>PyException_SetCause</a>  300 Objects/exceptions.c     PyException_SetCause(self, arg);
<a href='../S/3091.html#L3409'>PyException_SetCause</a> 3409 Python/ceval.c         PyException_SetCause(value, fixed_cause);
</pre>
</body>
</html>
